prusti-viper/src/encoder/procedure_encoder.rs

```diff
@@ -7008,7 +7017,7 @@ impl<'p, 'v: 'p, 'tcx: 'v> ProcedureEncoder<'p, 'v, 'tcx> {
         })
     }
 
-    fn register_error<T: Into<MultiSpan>>(&self, span: T, error_ctxt: ErrorCtxt) -> vir::Position {
+    pub fn register_error<T: Into<MultiSpan>>(&self, span: T, error_ctxt: ErrorCtxt) -> vir::Position {
```

Comment:
Where is this called from?

Reply:
In the body of `get_tick_call` in `encoder/resources/interface.rs`